schemas/content/product-price.schema.json

```diff
@@ -0,0 +1,50 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/content/product-price",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Product Price",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "Price model of a product in the product catalog.",
+  "definitions": {
+    "price": {
+      "properties": {
+        "xdm:amount": {
+          "title": "Amount",
+          "type": "number",
+          "minimum": 0,
```

Comment:
Are you sure about this? I'd assume you will find real data in PIMs that has negative amounts.

Reply:
I have seen what Lars allude to here in the real world. For example a discount as a product.

Reply:
I removed the minimum constraint, even I don't agree on negative prices. Using it for example to model discounts smells wrong to me, there are other/better ways to do that.